backend/aurcache/src/builder/makepkg_utils.rs

```diff
@@ -1,12 +1,8 @@
 pub fn create_makepkg_config(
-    name: String,
-    build_dir_base: &str,
+    _name: String,
+    _build_dir_base: &str,
```

Comment:
This is just temporary while we decide if we want this function at all in this state.

Reply:
Correct, we don't rely on any specific PKGDEST, we just look at the resulting archives in the current directory (like `makepkg` would do).